CLAUDE.md

```
@@ -190,6 +190,31 @@ Always add these attributes to interactive components for testability and debugg
 <Input id="pipeline-name" data-testid="pipeline-name-input" />
 ```
 
+### Component Attributes (Required)
```

markdownlint-cli2 (0.21.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)